### DISCUSSION TITLE
Preview transmog options before applying them

### DISCUSSION THREAD

**Discussion by bortage** [chromiecraft/chromiecraft#3573] [Suggestions]
There's an addon for the trinitycore version of the transmog plugin that this server uses where at the transmog vendor you can preview appearances before applying to your gear. if it's not difficult to implement it may be worth it to port that version of transmog over to this core

the trinitycore version can be found at https://github.com/Rochet2/TrinityCore/tree/transmogvendor_3.3.5/src/server/scripts/Custom/TransmogDisplayVendor
they have a preview video for the plugin in action here
https://www.youtube.com/watch?v=PIheEziN_dY

**Comment by Nyeriah** [comment]
Unfortunatelly, that's not an addon but an entirely different implementation of transmog and can't easily be implemented.

**Reply by Thunderbane** [reply]
Is it difficult to make items in the transmog window CTRL-clickable, so we can see the standard preview window for a single item just like it works for any such item in the inventory?
Transmog is such a great feature, but it becomes almost unusable when it lacks any preview..
